src/App.js

```diff
@@ -0,0 +1,15 @@
+/* eslint-disable import/no-extraneous-dependencies */
+import './App.css';
```

Comment:
- [ ] As I pointed out in the previous comments, Kindly make sure you fix your linters error instead of disabling them. This is for best practices reasons and clean code